go-controller/pkg/cni/cni.go

```diff
@@ -93,41 +93,47 @@ func (pr *PodRequest) checkOrUpdatePodUID(podUID string) error {
 	return nil
 }
 
-func (pr *PodRequest) getVFNetdevName() (string, error) {
-	// Get the vf device Name
-	vfNetdevices, err := util.GetSriovnetOps().GetNetDevicesFromPci(pr.CNIConf.DeviceID)
+func (pr *PodRequest) getNetdevName() (string, error) {
+	var netdevices []string
+	var err error
+	deviceID := pr.CNIConf.DeviceID
```

Comment:
@DmytroLinkin  Could you rebase and take into account the vDPA change?

Reply:
What about vDPA?

Reply:
I don't see it calls the vDPA library: https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/cni/cni.go#L97.

Reply:
Because vDPA PR got merged after my code.
I'll check what I need to change.